go.mod

```diff
@@ -6,67 +6,55 @@ require (
 	github.com/aws/aws-sdk-go v1.44.122
 	github.com/bramvdbogaerde/go-scp v1.2.0
 	github.com/google/uuid v1.3.0
-	github.com/hashicorp/terraform-plugin-docs v0.5.1
+	github.com/hashicorp/terraform-plugin-docs v0.16.0
 	github.com/hashicorp/terraform-plugin-log v0.7.0
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.10.1
 	github.com/yugabyte/platform-go-client v0.0.0-20230810122221-e283fa78cb44
-	golang.org/x/crypto v0.0.0-20220315160706-3147a52a75dd
+	golang.org/x/crypto v0.8.0
```

Comment:
Do we have to update all the imported modules to the latest version in this PR? Would it work just to update the terraform-plugin-docs version for now?

If the other updates are required, does it break any of the existing workflows?

Reply:
I did update but I didn't see any issue during the test .

Reply:
done, please check and resolve comment if ok